src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableTabletOpFlag.java

```diff
@@ -0,0 +1,61 @@
+/*-
+ * #%L
+ * com.oceanbase:obkv-table-client
+ * %%
+ * Copyright (C) 2021 - 2024 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute;
+
+public class ObTableTabletOpFlag {
+    private static final int FLAG_IS_SAME_TYPE             = 1 << 0;
+    private static final int FLAG_IS_SAME_PROPERTIES_NAMES = 1 << 1;
+    private long             flags                         = 0;
+
+    ObTableTabletOpFlag() {
+        setFlagIsSameType(true);
```

Comment:
Should we keep all elements false or 0, and set the corresponding value while using, in case we have too many elements and  need to remember the default value in the future

Reply:
Good idea !